tests/attestation_plugin/plugin/tests.h

```diff
@@ -22,6 +22,9 @@ void verify_sgx_evidence(
     size_t evidence_size,
     const uint8_t* endorsements,
     size_t endorsements_size,
+    bool null_endorsements, // if true, then use null endorsements for
+                            // oe_verify_evidence(), original endorsements is
+                            // used to check endorsement claim
```

Comment:
Instead of `null_endorsements`, how about passing in `expected_claims` and `expected_claims_size`? When a non-empty list of expected claims are passed in, `verigy_sgx_evidence()` can verify that `oe_verify_evidence()` gets all the expected claims.

Reply:
Thanks for suggestion!
Yes, that makes code cleaner and easier to understand.

Reply:
I suggested passing in expected claims directly, the caller would be responsible to put together the list of claims. I see that your updated implementation passes in expected SGX endorsements instead. I think it's not as flexible, but still  works.

Reply:
Yes, I've thought of passing in the expected claims but found the contents of claims like OE_REQUIRED_CLAIMS are embedded in the quote report data inside the evidence. Extracting them would be tricky and duplicated with unwrapping work done in the `oe_verify_evidence()`. So I left them be in previous way.
And also try to save conversion from endorsements to claims (name-value pair).